novawallet/Common/Services/Web3AlertService/RemotePushSettings.swift

```diff
@@ -0,0 +1,78 @@
+import FirebaseCore
+import FirebaseFirestore
+import SubstrateSdk
+import RobinHood
+
+typealias ChainSelection = RemotePushSettings.Selection<[String]>
```

Comment:
I would either name it RemotePushChainSelection or put inside RemotePushSettings. Because now it is quite hard to guess meaning of the type.